inst/qml/qml_components/CochraneInput.qml

```diff
@@ -156,8 +92,54 @@ Section
 		textType:	JASP.TextTypeSource
 	}
 
+
+	RadioButtonGroup
+	{
+		columns:			3
+		name:				"selectionType"
+		title:				qsTr("Select systematic reviews based on")
+		id:					dataType
+
+		RadioButton
+		{
+			value:		"selectionTopics"
+			label:		qsTr("Topics")
+			id:			selectionTopics
+			checked:	true
+		}
+
+		RadioButton
+		{
+			value:		"selectionKeywords"
+			label:		qsTr("Keywords")
+			id:			selectionKeywords
+		}
+
+		RadioButton
+		{
+			value:		"selectionTextSearch"
+			label:		qsTr("Search titles")
+			id:			selectionTextSearch
+		}
+	}
+
+	DropDown
+	{
+		name:				"analyzeAs"
+		label:				qsTr("Analyze as")
+		visible:			analysisType === "classicalDichotomous" || analysisType === "bayesianDichotomous"
+		values:
+		[
+			{label: qsTr("Log(odds ratios)"),			value: "OR"},
+			{label: qsTr("Log(Peto's odds ratios)"),	value: "POR"},
+			{label: qsTr("Log(risk ratios)"),			value: "RR"},
+			{label: qsTr("Risk differences"),			value: "RD"	}
+		]
+	}
+
 	ComponentsList
 	{
+		title:				qsTr("Selected Reviews/Meta-Analyses")
```

Comment:
Can you make this title and the title to `selectionType` ("Select systematic reviews based on") consistent? I assume it's either both

"Selecting systematic review/meta-analyses based on" + "Selected systematic reviews/meta-analyses"
or
"Selecting Reviews/Meta-Analyses based on" + "Selected Reviews/Meta-Analyses"

or something like that... I hope you get the gist